build.rs

```diff
@@ -65,7 +65,7 @@
 )]
 
 use std::env;
-use std::path::Path;
+use std::path::{Path, PathBuf};
```

Comment:
There are some uses of `std::path::PathBuf` below that should have the `std::path::` qualification removed.